pkg/verifiedcar/verifiedcar.go

```diff
@@ -55,8 +55,6 @@ type Config struct {
 	MaxBlocks          uint64         // set a budget for the traversal
```

Comment:
eek so, I just realized that we're gonna need to figure out what to do with .storage, as they are not yet supporting ranges. I can pop over and try to add it but... in the meantime, maybe we need to figure out some content negotiation in the http retriever.

Reply:
otherwise we're looking at 100% failure here.

Reply:
Not 100%. Bitswap will work, Frisbii will work, but yeah the rest will bork if we start pushing them through.

I had some thoughts about negotiation over in 412: https://github.com/ipfs/specs/pull/412#discussion_r1247310092

If we were to request a byte range but then be told the server doesn't support it, at least we could switch modes to parsing their response. What we do on the way out of Lassie is another matter—do we give the user more than they asked for and just give our own negotiation signal that we don't support range cause we couldn't find anyone else that does? Either way this is really hard.

Reply:
technically if we knew that we were getting a full response, we could modify the verifier to just skip blocks till you got to the expected blocks in a range request (since technically a full request contains all the blocks for a range request in the correct order, just with other blocks in between). In fact, turning this on when entity-bytes is present by default might be the hack needed to get this over the line.